src/main/java/com/ibm/nfvodriver/service/LifecycleManagementService.java

```diff
@@ -37,44 +37,44 @@ public LifecycleManagementService(NSLifecycleManagementDriver nsLifecycleManagem
+            final String requestId = UUID.randomUUID().toString();
+            // Delay sending the asynchronous response (from a different thread) as this method needs to complete first (to send the response back to Brent)
+            externalMessagingService.sendDelayedExecutionAsyncResponse(new ExecutionAsyncResponse(requestId, ExecutionStatus.COMPLETE, null, outputs, Collections.emptyMap()), properties.getExecutionResponseDelay());
+
+            return new ExecutionAcceptedResponse(requestId);
+
+            } else if ("Install".equalsIgnoreCase(executionRequest.getLifecycleName())) {
+                // Instantiate
+                 final String nsInstanceId = executionRequest.getStringResourceProperty("nsInstanceId");
+                 final String instantiateNsRequest = messageConversionService.generateMessageFromRequest("InstantiateNsRequest", executionRequest);
+                 final String requestId = nsLifecycleManagementDriver.instantiateNs(executionRequest.getDeploymentLocation(), nsInstanceId, instantiateNsRequest);
+                 return new ExecutionAcceptedResponse(requestId);
 
-                // Send response back to ALM
-                return new ExecutionAcceptedResponse(requestId);
             } else if ("Configure".equalsIgnoreCase(executionRequest.getLifecycleName())) {
                 // Instantiate
                 final String nsInstanceId = executionRequest.getStringResourceProperty("nsInstanceId");
                 final String instantiateNsRequest = messageConversionService.generateMessageFromRequest("InstantiateNsRequest", executionRequest);
                 final String requestId = nsLifecycleManagementDriver.instantiateNs(executionRequest.getDeploymentLocation(), nsInstanceId, instantiateNsRequest);
                 return new ExecutionAcceptedResponse(requestId);
 
-            } else if ("Start".equalsIgnoreCase(executionRequest.getLifecycleName())) {
-                // Operate (Start)
+            } else if ("Update".equalsIgnoreCase(executionRequest.getLifecycleName())) {
+                // Operate (Update)
                 final String nsInstanceId = executionRequest.getStringResourceProperty("nsInstanceId");
-                final String updateNsRequest = messageConversionService.generateMessageFromRequest("UpdateNsRequest-Start", executionRequest);
+                final String updateNsRequest = messageConversionService.generateMessageFromRequest("UpdateNsRequest", executionRequest);
                 final String requestId = nsLifecycleManagementDriver.updateNs(executionRequest.getDeploymentLocation(), nsInstanceId, updateNsRequest);
                 return new ExecutionAcceptedResponse(requestId);
-            }  else if ("Stop".equalsIgnoreCase(executionRequest.getLifecycleName())) {
-                // Operate (Stop)
+            } else if ("Delete".equalsIgnoreCase(executionRequest.getLifecycleName())) {
+                // Operate (Delete)
```

Comment:
Remove Operate and just keep Delete in the commented line

Reply:
updated the code and committed